bom/pom.xml

```diff
@@ -34,6 +34,7 @@
     <dispatch-router.image.name>quay.io/enmasse/qdrouterd-base:1.7.0</dispatch-router.image.name>
     <grafana.version>5.3.2</grafana.version>
     <guava.version>25.0-jre</guava.version>
+    <caffeine.version>2.7.0</caffeine.version>
```

Comment:
the idea is to have the list in alphabetical order ...
BTW does it also work with version 2.6.2? That would be easier to get approved ...

Reply:
It works with version 2.6.2. I will push the changes.